db/upgrade.php

```diff
@@ -57,5 +57,42 @@ function xmldb_block_acclaim_upgrade($oldversion) {
         // Acclaim savepoint reached.
         upgrade_block_savepoint(true, 2020042200, 'acclaim');
     }
+
+    if ($oldversion < 2024013100) {
```

Comment:
Just for my own knowledge where do you get these version numbers?

Reply:
It's usually just the current date you're making the change. Think of it like how our rails migrations append a date to the file—moodle tracks plugin schema revisions in a similar manner.